packages/backend/src/controllers/admin.ts

```diff
@@ -0,0 +1,57 @@
+import { NextFunction, Request, Response, Router } from "express";
+import { Queue } from "../lib/models/queue";
+import { HttpException } from "../lib/errors";
+import { GETQueueReq, GETQueueRes } from "@waiting-line-app/shared-dto/admin";
+import { IQueue, IAdmin } from "@waiting-line-app/shared-dto/db";
+
+const getAdminQueue = async (adminId: string) => {
+  const admin: IAdmin = await Queue.find({ adminId });
+  if (!admin) {
+    throw new Error("admin not found");
+  }
+  const arrOfQueueProperties: IQueue = [];
+  for (let i = 0; i < admin.length; i++) {
+    const queueName: string = admin[i].queueName;
```

Comment:
Typescript implicitly types! Please do not explicitly write the types if you do not have to. (It's not like java)

Reply:
Which line are you referring to? When I hovered my mouse over `queueName = admin[i].queueName`, queueName was type any. But I'll change it back